src/configuration/index.ts

```diff
@@ -12,22 +11,18 @@ const mainnetBasicController = "0xa05Ae774Da859943B7B859cd2A6aD9F5f1651d6a";
 export const mainnetConfiguration = (
   provider: ethers.providers.Provider
 ): Config => {
+  const mainnetConfig: zAuctionConfig = {
+    web3Provider: provider,
+    network: "rinkeby",
```

Comment:
Still looks like the wrong network? This is mainnet config, why are we using rinkeby network?